session_utils/wscript

```diff
@@ -86,7 +86,7 @@ def build(bld):
     utils = bld.path.ant_glob('*.cc', excl=['example.cc', 'common.cc'])
 
     for util in utils:
-        fn = os.path.basename(str(util))[:-3]
+        fn = os.path.splitext(os.path.basename(str(util)))[0]
```

Comment:
session-utils must be C++ files with a "`.cc`" extension and compiled as such (see session_utils/README).

The `ant_glob` three lines above will only provide files with that end with ".cc". So it can never happen that the file extension will be something other than precisely 3 chars `.cc`.

That being said,  stripping the extension is fine regardless.